inbm/cloudadapter-agent/proto/inbc-mock/README.md

```diff
@@ -0,0 +1,9 @@
+# inbc-mock
+
+This serves as a mock INBC that can be used to send ping requests
+to cloudadapter. The main loop will ping cloudadapter once per second.
+
+Prerequisites:
+
+* go 1.20 or higher
+* protobuf tools installed for regenerating protobuf golang code
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous$</h3>

Expand the README to include instructions on how to regenerate protobuf golang code after modifications.

Reply:
Good point, need a reference to the protobuf generation shell script here